docs/book/src/topics/aso.md

```diff
@@ -56,13 +56,26 @@ Cluster API.
 
 ### Installing more CRDs
 
-CAPZ's installation of ASO configures only the ASO CRDs that are required by CAPZ. To make more resource types
-available, install their corresponding CRDs. ASO publishes a manifest containing all CRDs for each
-[release](https://github.com/Azure/azure-service-operator/releases). Extract only the ones you need using tool
-like [`yq`](https://mikefarah.gitbook.io/yq/), then make the following modifications to each CRD to account
-for CAPZ installing ASO in the `capz-system` namespace:
-
-- Change `metadata.annotations."cert-manager.io/inject-ca-from"` to `capz-system/azureserviceoperator-serving-cert`
-- Change `spec.conversion.webhook.clientConfig.service.namespace` to `capz-system`
+#### For a fresh installation
+From the release v1.13.2, users can specify additional ASO CRDs to be installed in the management cluster by exporting `ADDITIONAL_ASO_CRDS` variable.
```

Comment:
Nit: I don't think we need to worry about specifying the release version here, as the netlify publishing produces doc artifacts per-release.

I also don't think we want to include this change into a CAPZ patch release (we'd ship this w/ v1.14.0)

cc @mboersma for sanity checking the netlify statement
cc @dtzar about the patch release statement

Reply:
Agree, we should remove the version in the docs.  Also think this should be included with 1.14, not the patch.  Re: SemVer - this is essentially a new feature, not a patch.

Reply:
Netlify publishes from the latest cut release branch, so you're right that any mention of it here won't appear on the site until we cut Netlify over to the release-1.14 branch.